articles/search/search-indexer-field-mappings.md

```diff
@@ -290,4 +290,24 @@ When you retrieve the encoded key at search time, you can then use the `urlDecod
       "name" : "urlDecode"
     }
   }]
- ```
\ No newline at end of file
+ ```
+ 
+ ### fixedLengthEncode function
+ 
+ This function converts a string of any length to a fixed length string.
+ 
+ ### Example - map document keys that are too long
+ 
+When facing errors complaining about document key being longer than 1024 characters, this function can be applied to reduce the length of the document key.
```

Comment:
Add a line explaining that keys must be at most 1024 characters.Probably modifying the existing base64 documentation

> Only URL-safe characters can appear in an Azure Cognitive Search document key (because customers must be able to address the document using the Lookup API ). If the source field for your key contains URL-unsafe characters, you can use the base64Encode function to convert it at indexing time.